models/public/mobilefacedet-v1-mxnet/quantization.yml

```diff
@@ -0,0 +1,10 @@
+compression:
+  algorithms:
+    - name: DefaultQuantization
+      params:
+        num_samples_for_tuning: 2000
+        preset: accuracy
+        stat_subset_size: 300
+        use_fast_bias: False
+        use_layerwise_tuning: False
+  dump_intermediate_model: True
```

Comment:
Is this option needed? I don't really know what it does, but from the name it doesn't seem like it would affect the quantization result.

Reply:
Removed